Freeradius3 because openssl 1.1.1 Error

Maintainer: @LucileQ
Environment: mipsel_24kc, R3G, OpenWrt master
                      mvebu, EBIN, OpenWrt master

**The following error occurred while using official build**
```
Collected errors:
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for freeradius3:
 *      libopenssl1.0.0
 * opkg_install_cmd: Cannot install package freeradius3.

root@OpenWrt:~# radiusd -X
Error loading shared library libcrypto.so.1.0.0: No such file or directory (needed by /usr/sbin/radiusd)
Error loading shared library libssl.so.1.0.0: No such file or directory (needed by /usr/sbin/radiusd)
Error loading shared library libtalloc.so.2: No such file or directory (needed by /usr/sbin/radiusd)
Error loading shared library libreadline.so.8: No such file or directory (needed by /usr/sbin/radiusd)
Error loading shared library libcrypto.so.1.0.0: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-server.so)
Error loading shared library libssl.so.1.0.0: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-server.so)
Error loading shared library libcrypto.so.1.0.0: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-radius.so)
Error loading shared library libssl.so.1.0.0: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-radius.so)
Error loading shared library libtalloc.so.2: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-radius.so)
Error loading shared library libreadline.so.8: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-radius.so)
Error loading shared library libpcap.so.1: No such file or directory (needed by /usr/lib/freeradius3/libfreeradius-radius.so)
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_get_size: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_steal_loc: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_set_destructor: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_version_minor: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: SSLeay_version: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_realloc_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_parent: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_zero: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_free: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_vasprintf: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_init: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_version_major: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_get_type_abort: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: SSLeay: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_strndup: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_strdup: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_set_name_const: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_memdup: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_strdup_append: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_zero_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_asprintf: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: talloc_named_const: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-server.so: _talloc_realloc: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_set_name_const: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_asprintf: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_set_buffer_size: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_disable_null_tracking: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_set_destructor: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_lookupnet: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: MD5_Update: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_dump_open: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_realloc_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_datalink: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_open_live: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_zero: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: MD5_Init: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_named_const: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_set_log_fn: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_dump_close: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_vasprintf: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_set_abort_fn: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_parent: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_memdup: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_get_size: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_compile: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_steal_loc: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_free: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_open_offline: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: MD4_Final: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_set_promisc: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_dump_flush: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_geterr: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_fopen_offline: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: MD4_Init: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_report_full: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_set_snaplen: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: MD4_Update: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_open_dead: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: HMAC_CTX_cleanup: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: HMAC_CTX_init: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_get_type_abort: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_close: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_dump_fopen: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_create: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: _talloc_zero_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_get_selectable_fd: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_set_timeout: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_strdup: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_autofree_context: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: CRYPTO_malloc: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: MD5_Final: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_setfilter: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_activate: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: CRYPTO_free: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: talloc_get_name: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-radius.so: pcap_setnonblock: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: HMAC_Update: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: SSL_SESSION_get_ex_data: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: _talloc_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: talloc_named_const: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: _talloc_zero_array: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: EVP_sha1: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: SSL_export_keying_material: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: talloc_get_size: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: HMAC_CTX_set_flags: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: EVP_md5: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: HMAC_Init_ex: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: HMAC_Final: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: EVP_MD_size: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: SSL_get_session: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: _talloc_free: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: _talloc_steal_loc: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: SSL_state: symbol not found
Error relocating /usr/lib/freeradius3/libfreeradius-eap.so: SSL_set_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_response_get1_basic: symbol not found
Error relocating /usr/sbin/radiusd: talloc_set_name_const: symbol not found
Error relocating /usr/sbin/radiusd: talloc_set_name: symbol not found
Error relocating /usr/sbin/radiusd: talloc_asprintf: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_bio: symbol not found
Error relocating /usr/sbin/radiusd: talloc_disable_null_tracking: symbol not found
Error relocating /usr/sbin/radiusd: EC_KEY_new_by_curve_name: symbol not found
Error relocating /usr/sbin/radiusd: SSL_get_rbio: symbol not found
Error relocating /usr/sbin/radiusd: BIO_s_mem: symbol not found
Error relocating /usr/sbin/radiusd: BIO_new_file: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_set_destructor: symbol not found
Error relocating /usr/sbin/radiusd: SSL_get_ex_data_X509_STORE_CTX_idx: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_array: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_CTX_set_error: symbol not found
Error relocating /usr/sbin/radiusd: CRYPTO_set_locking_callback: symbol not found
Error relocating /usr/sbin/radiusd: BIO_write: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_REQ_CTX_set1_req: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_CTX_get_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_psk_server_callback: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_check_private_key: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_zero: symbol not found
Error relocating /usr/sbin/radiusd: BIO_puts: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_get_cert_store: symbol not found
Error relocating /usr/sbin/radiusd: SSL_alert_desc_string_long: symbol not found
Error relocating /usr/sbin/radiusd: talloc_named_const: symbol not found
Error relocating /usr/sbin/radiusd: X509V3_EXT_d2i: symbol not found
Error relocating /usr/sbin/radiusd: talloc_init: symbol not found
Error relocating /usr/sbin/radiusd: BIO_free_all: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_info_callback: symbol not found
Error relocating /usr/sbin/radiusd: ERR_get_error: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_REQUEST_new: symbol not found
Error relocating /usr/sbin/radiusd: talloc_enable_null_tracking: symbol not found
Error relocating /usr/sbin/radiusd: SSL_load_error_strings: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_free: symbol not found
Error relocating /usr/sbin/radiusd: SSL_connect: symbol not found
Error relocating /usr/sbin/radiusd: SSL_get_session: symbol not found
Error relocating /usr/sbin/radiusd: sk_num: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_request_add1_nonce: symbol not found
Error relocating /usr/sbin/radiusd: CRYPTO_THREADID_set_callback: symbol not found
Error relocating /usr/sbin/radiusd: talloc_vasprintf: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_check_validity: symbol not found
Error relocating /usr/sbin/radiusd: EC_KEY_free: symbol not found
Error relocating /usr/sbin/radiusd: talloc_free_children: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_new: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_verify_depth: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_cipher_list: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_verify: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_use_PrivateKey_file: symbol not found
Error relocating /usr/sbin/radiusd: talloc_parent: symbol not found
Error relocating /usr/sbin/radiusd: SSL_pending: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_set_flags: symbol not found
Error relocating /usr/sbin/radiusd: SSL_alert_type_string_long: symbol not found
Error relocating /usr/sbin/radiusd: SSL_peek: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_CTX_get0_current_issuer: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_CTX_get_error_depth: symbol not found
Error relocating /usr/sbin/radiusd: talloc_get_size: symbol not found
Error relocating /usr/sbin/radiusd: SSL_load_client_CA_file: symbol not found
Error relocating /usr/sbin/radiusd: OBJ_obj2txt: symbol not found
Error relocating /usr/sbin/radiusd: SSL_ctrl: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_sendreq_nbio: symbol not found
Error relocating /usr/sbin/radiusd: X509_get_ext_d2i: symbol not found
Error relocating /usr/sbin/radiusd: CRYPTO_THREADID_set_numeric: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_cert_status_str: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_steal_loc: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_load_verify_locations: symbol not found
Error relocating /usr/sbin/radiusd: BIO_free: symbol not found
Error relocating /usr/sbin/radiusd: BIO_ctrl: symbol not found
Error relocating /usr/sbin/radiusd: BIO_new: symbol not found
Error relocating /usr/sbin/radiusd: BIO_new_connect: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_free: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_accept_state: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_new: symbol not found
Error relocating /usr/sbin/radiusd: BIO_test_flags: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_RESPONSE_free: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_response_status_str: symbol not found
Error relocating /usr/sbin/radiusd: PEM_write_X509: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_free: symbol not found
Error relocating /usr/sbin/radiusd: DH_free: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_parse_url: symbol not found
Error relocating /usr/sbin/radiusd: ERR_free_strings: symbol not found
Error relocating /usr/sbin/radiusd: GENERAL_NAMES_free: symbol not found
Error relocating /usr/sbin/radiusd: ERR_clear_error: symbol not found
Error relocating /usr/sbin/radiusd: EVP_cleanup: symbol not found
Error relocating /usr/sbin/radiusd: SSL_state_string_long: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_basic_verify: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_sendreq_new: symbol not found
Error relocating /usr/sbin/radiusd: X509_EXTENSION_get_object: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_crl_reason_str: symbol not found
Error relocating /usr/sbin/radiusd: SSL_shutdown: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_CTX_get_error: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_use_certificate_file: symbol not found
Error relocating /usr/sbin/radiusd: BIO_ctrl_pending: symbol not found
Error relocating /usr/sbin/radiusd: OPENSSL_add_all_algorithms_noconf: symbol not found
Error relocating /usr/sbin/radiusd: X509V3_EXT_print: symbol not found
Error relocating /usr/sbin/radiusd: SSL_new: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_ctrl: symbol not found
Error relocating /usr/sbin/radiusd: X509_get_serialNumber: symbol not found
Error relocating /usr/sbin/radiusd: X509_get_ext_by_NID: symbol not found
Error relocating /usr/sbin/radiusd: PEM_read_bio_DHparams: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_CTX_get_current_cert: symbol not found
Error relocating /usr/sbin/radiusd: ERR_error_string_n: symbol not found
Error relocating /usr/sbin/radiusd: X509_verify_cert_error_string: symbol not found
Error relocating /usr/sbin/radiusd: SSL_read: symbol not found
Error relocating /usr/sbin/radiusd: EXTENDED_KEY_USAGE_free: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: ASN1_GENERALIZEDTIME_print: symbol not found
Error relocating /usr/sbin/radiusd: SSL_library_init: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_resp_find_status: symbol not found
Error relocating /usr/sbin/radiusd: CONF_modules_unload: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_cert_to_id: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: SSL_SESSION_get_ex_new_index: symbol not found
Error relocating /usr/sbin/radiusd: SSL_get_psk_identity: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_get_type_abort: symbol not found
Error relocating /usr/sbin/radiusd: SSL_SESSION_get_id: symbol not found
Error relocating /usr/sbin/radiusd: X509_get_issuer_name: symbol not found
Error relocating /usr/sbin/radiusd: X509_get_subject_name: symbol not found
Error relocating /usr/sbin/radiusd: CRYPTO_num_locks: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_REQ_CTX_free: symbol not found
Error relocating /usr/sbin/radiusd: ASN1_STRING_data: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_default_passwd_cb_userdata: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_default_passwd_cb: symbol not found
Error relocating /usr/sbin/radiusd: _talloc_zero_array: symbol not found
Error relocating /usr/sbin/radiusd: X509_get_ext: symbol not found
Error relocating /usr/sbin/radiusd: BIO_read: symbol not found
Error relocating /usr/sbin/radiusd: X509_NAME_oneline: symbol not found
Error relocating /usr/sbin/radiusd: talloc_strdup: symbol not found
Error relocating /usr/sbin/radiusd: talloc_pool: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_verify: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_fd: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_quiet_shutdown: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_check_nonce: symbol not found
Error relocating /usr/sbin/radiusd: ERR_get_error_line: symbol not found
Error relocating /usr/sbin/radiusd: ENGINE_cleanup: symbol not found
Error relocating /usr/sbin/radiusd: CRYPTO_cleanup_all_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_use_certificate_chain_file: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_response_status: symbol not found
Error relocating /usr/sbin/radiusd: X509_STORE_load_locations: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_request_add0_id: symbol not found
Error relocating /usr/sbin/radiusd: SSL_want: symbol not found
Error relocating /usr/sbin/radiusd: ERR_print_errors: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_BASICRESP_free: symbol not found
Error relocating /usr/sbin/radiusd: SSL_free: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_psk_client_callback: symbol not found
Error relocating /usr/sbin/radiusd: SSLv23_method: symbol not found
Error relocating /usr/sbin/radiusd: sk_value: symbol not found
Error relocating /usr/sbin/radiusd: SSL_set_msg_callback: symbol not found
Error relocating /usr/sbin/radiusd: SSL_get_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: ERR_remove_thread_state: symbol not found
Error relocating /usr/sbin/radiusd: ERR_peek_error: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_REQ_CTX_add1_header: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_info_callback: symbol not found
Error relocating /usr/sbin/radiusd: SSL_write: symbol not found
Error relocating /usr/sbin/radiusd: OBJ_obj2nid: symbol not found
Error relocating /usr/sbin/radiusd: RAND_load_file: symbol not found
Error relocating /usr/sbin/radiusd: SSL_SESSION_set_ex_data: symbol not found
Error relocating /usr/sbin/radiusd: OBJ_sn2nid: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_remove_session: symbol not found
Error relocating /usr/sbin/radiusd: CONF_modules_load_file: symbol not found
Error relocating /usr/sbin/radiusd: BIO_new_fp: symbol not found
Error relocating /usr/sbin/radiusd: i2a_ASN1_OBJECT: symbol not found
Error relocating /usr/sbin/radiusd: OCSP_REQUEST_free: symbol not found
Error relocating /usr/sbin/radiusd: SSL_get_error: symbol not found
Error relocating /usr/sbin/radiusd: SSL_state: symbol not found
Error relocating /usr/sbin/radiusd: SSL_CTX_set_client_CA_list: symbol not found
Error relocating /usr/sbin/radiusd: X509_NAME_get_text_by_NID: symbol not found
 ```

**Build error when I build**
```
src/main/tls.c: In function 'tls_global_cleanup':
src/main/tls.c:2680:2: warning: implicit declaration of function 'ENGINE_cleanup'; did you mean 'EVP_PBE_cleanup'? [-Wimplicit-function-declaration]
  ENGINE_cleanup();
  ^~~~~~~~~~~~~~
  EVP_PBE_cleanup
src/main/tls.c: In function 'tls_init_ctx':
src/main/tls.c:3198:9: warning: implicit declaration of function 'RAND_load_file'; did you mean 'CONF_load_bio'? [-Wimplicit-function-declaration]
   if (!(RAND_load_file(conf->random_file, 1024*10))) {
         ^~~~~~~~~~~~~~
         CONF_load_bio
CC src/main/conffile.c
CC src/main/evaluate.c
CC src/main/exec.c
CC src/main/exfile.c
CC src/main/log.c
CC src/main/parser.c
CC src/main/map.c
CC src/main/regex.c
CC src/main/tmpl.c
CC src/main/util.c
CC src/main/pair.c
CC src/main/xlat.c
/home/nasbdh9/openwrt/staging_dir/toolchain-mipsel_24kc_gcc-7.4.0_musl/lib/gcc/mipsel-openwrt-linux-musl/7.4.0/../../../../mipsel-openwrt-linux-musl/bin/ld: /home/nasbdh9/openwrt/build_dir/target-mipsel_24kc_musl/freeradius-server-release_3_0_17/build/objs/src/main/tls.o: in function `tls_global_cleanup':
tls.c:(.text+0x2ef6): undefined reference to `ENGINE_cleanup'
collect2: error: ld returned 1 exit status
scripts/boiler.mk:628: recipe for target '/home/nasbdh9/openwrt/build_dir/target-mipsel_24kc_musl/freeradius-server-release_3_0_17/build/bin/local/radeapclient' failed
make[4]: *** [/home/nasbdh9/openwrt/build_dir/target-mipsel_24kc_musl/freeradius-server-release_3_0_17/build/bin/local/radeapclient] Error 1
make[4]: Leaving directory '/home/nasbdh9/openwrt/build_dir/target-mipsel_24kc_musl/freeradius-server-release_3_0_17'
Makefile:607: recipe for target '/home/nasbdh9/openwrt/build_dir/target-mipsel_24kc_musl/freeradius-server-release_3_0_17/.built' failed
make[3]: *** [/home/nasbdh9/openwrt/build_dir/target-mipsel_24kc_musl/freeradius-server-release_3_0_17/.built] Error 2
make[3]: Leaving directory '/home/nasbdh9/openwrt/feeds/packages/net/freeradius3'
time: package/feeds/packages/freeradius3/compile#14.92#2.88#17.89
package/Makefile:107: recipe for target 'package/feeds/packages/freeradius3/compile' failed
make[2]: *** [package/feeds/packages/freeradius3/compile] Error 2
make[2]: Leaving directory '/home/nasbdh9/openwrt'
package/Makefile:103: recipe for target '/home/nasbdh9/openwrt/staging_dir/target-mipsel_24kc_musl/stamp/.package_compile' failed
make[1]: *** [/home/nasbdh9/openwrt/staging_dir/target-mipsel_24kc_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/nasbdh9/openwrt'
/home/nasbdh9/openwrt/include/toplevel.mk:216: recipe for target 'world' failed
make: *** [world] Error 2
```

--- a/net/freeradius3/Makefile
+++ b/net/freeradius3/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=freeradius3
 PKG_VERSION:=release_3_0_17
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://github.com/FreeRADIUS/freeradius-server/archive
@@ -380,6 +380,13 @@ define Package/freeradius3-utils
   DEPENDS:=+freeradius3-common
   TITLE:=Misc. client utilities
 endef
+
+# This is a workaround for a bug in configure.ac, that does not check
+# for the openss/rand.h header.  Setting the macro is safe for openwrt,
+# and faster than rebuilding configure.
+# This has been fixed upstream in the 3.0.x branch, and should not be
+# needed in a future release.
+EXTRA_CFLAGS+= -DHAVE_OPENSSL_RAND_H
 
 CONFIGURE_ARGS+= \
 	--libdir=/usr/lib/freeradius3 \

--- a/net/freeradius3/patches/004-tls.c-allow-build-with-no-openssl-engine-support.patch
+++ b/net/freeradius3/patches/004-tls.c-allow-build-with-no-openssl-engine-support.patch
@@ -1,0 +1,24 @@
+From 0b17cf6e39064b008792811a6babf4cd75ac7744 Mon Sep 17 00:00:00 2001
+From: Eneas U de Queiroz <cote2004-github@yahoo.com>
+Date: Mon, 25 Feb 2019 10:48:48 -0300
+Subject: [PATCH] tls.c: allow build with no openssl engine support
+
+This avoids an ENGINE_cleanup call if openssl was compiled without
+engine support.
+
+Signed-off-by: Eneas U de Queiroz <cote2004-github@yahoo.com>
+
+diff --git a/src/main/tls.c b/src/main/tls.c
+index 73f0e3a3e8..9726953234 100644
+--- a/src/main/tls.c
++++ b/src/main/tls.c
+@@ -2743,7 +2743,9 @@ void tls_global_cleanup(void)
+ #elif OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
+ 	ERR_remove_thread_state(NULL);
+ #endif
++#ifndef OPENSSL_NO_ENGINE
+ 	ENGINE_cleanup();
++#endif
+ 	CONF_modules_unload(1);
+ 	ERR_free_strings();
+ 	EVP_cleanup();

